runtime/hydra-dx/src/lib.rs

```diff
@@ -820,6 +813,39 @@ impl pallet_scheduler::Config for Runtime {
 	type WeightInfo = ();
 }
 
+pub struct OnlyGalacticCouncil;
+impl EnsureOrigin<Origin> for OnlyGalacticCouncil {
+	type Success = AccountId;
+
+	fn try_origin(o: Origin) -> Result<Self::Success, Origin> {
+		Into::<Result<RawOrigin<AccountId>, Origin>>::into(o).and_then(|o| match o {
+			RawOrigin::Signed(caller) => {
+				if caller == hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into() {
+					Ok(caller)
+				} else {
+					Err(Origin::from(Some(caller)))
+				}
+			}
+			r => Err(Origin::from(r)),
+		})
+	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn successful_origin() -> Origin {
+		Origin::from(RawOrigin::Signed(Default::default()))
+	}
+}
+
+impl orml_vesting::Config for Runtime {
+	type Event = Event;
+	type Currency = Balances;
+	type MinVestedTransfer = MinVestedTransfer;
+	type VestedTransferOrigin = OnlyGalacticCouncil;
```

Comment:
root or galactic council

Reply:
AFAIK we can't use the `root` account here because the implementation of `VestedTransferOrigin` requires `AccountId` as a return type and it is something what the `root` account doesn't have https://docs.substrate.io/rustdocs/latest/frame_system/enum.RawOrigin.html.

Reply:
You could call it with Galactic account in case origin is Root.

Reply:
Do you mean something like this?
```
impl EnsureOrigin<Origin> for GalacticCouncilOrRoot {
	type Success = AccountId;

	fn try_origin(o: Origin) -> Result<Self::Success, Origin> {
		Into::<Result<RawOrigin<AccountId>, Origin>>::into(o).and_then(|o| match o {
			RawOrigin::Signed(caller) => {
				if caller == hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into() {
					Ok(caller)
				} else {
					Err(Origin::from(Some(caller)))
				}
			}
			RawOrigin::Root => Ok(hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into()),
			r => Err(Origin::from(r)),
		})
	}
```

Reply:
yup

Reply:
i would also rather use constant everywhere than accountid value directly